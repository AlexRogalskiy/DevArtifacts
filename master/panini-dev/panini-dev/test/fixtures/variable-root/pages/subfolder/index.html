<img src="{{root}}image.jpg">
{{> partial}}
