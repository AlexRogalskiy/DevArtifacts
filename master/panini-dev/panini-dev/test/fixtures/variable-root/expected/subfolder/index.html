<html>
  <body>
    <img src="../image.jpg">
    <img src="../partial.png">
    
  </body>
</html>
